tests/cross_functional/system_test/test_nfs_inout_export.py

```
@@ -0,0 +1,1188 @@
+
+        # Wait for NFS server pod to be running again
+        log.info("Waiting for NFS server pod to be running...")
+        wait_for_pods_to_be_running(
+            pod_names=[nfs_server_pod.name], namespace=self.namespace, timeout=600
+        )
+        log.info("NFS server pod is running again")
+
+        # Verify NFS mount is still accessible
+        log.info("Verifying NFS mount accessibility...")
+        retcode, stdout, _ = con.exec_cmd(f"findmnt -M {test_folder_for_pod}")
+        assert retcode == 0, f"NFS mount not accessible after node reboot"
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)